src/components/header/header.scss

```diff
@@ -0,0 +1,4 @@
+.MuiToolbar-root {
+  padding-left: 0 !important;
+  padding-right: 0 !important;
```

Comment:
try to avoid important, maybe create a more specific selector

Reply:
Fixed it.